locales/en/messages.json

```diff
@@ -4920,7 +4920,7 @@
         "description": "One of the elements of the OSD"
     },
     "osdDescElementCraftName": {
-        "message": "Craft name as set in Configuration tab"
```

Comment:
Question to the reviewers:

Given that I changed some of the current `en` texts - should I do anything extra for all other translations?

Looking at https://github.com/betaflight/betaflight-configurator/pull/2791 I _suspect_ I might have to manually remove them for now from all language files (so they can be re-translated in the future).

Reply:
No, this will be done automatically. Crowdin will delete the translations in the next language files update.